pkg/api/configsync/v1beta1/constants.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1beta1
+
+import (
+	"kpt.dev/configsync/pkg/api/configmanagement"
+	"kpt.dev/configsync/pkg/reconcilermanager"
+)
+
+// SyncFinalizer is a finalizer handled by ReconcilerManager to ensure
+// deletion of managed resource objects complete before the RootSync/RepoSync is
+// deleted
+const SyncFinalizer = reconcilermanager.ManagerName + "." + configmanagement.GroupName
```

Comment:
Change annotation to be consistent with the reconciler finalizer domain

Reply:
And rename to include ReconcilerManager.
And define in the same place with the ReconcilerFinalizer

Reply:
configsync.gke.io/reconciler-manager